src/Cmfd.cpp

```diff
@@ -905,11 +905,11 @@ void Cmfd::initializeCurrents() {
 
   /* Delete old Cmfd surface currents array it it exists */
```

Comment:
While you are fixing this part of the code, can you also correct the comment here to fix this typo and the one for the corner currents?

it it -> if it